Conference Banner and Participation Update (#15)
USRSE/usrse24: pages/participate/participate.md

- Training and workforce development
- Maturing and expanding the RSE profession

> **Submission Website**: [EasyChair](https://easychair.org/conferences/?conf=usrse2024)

Reviewer: ```suggestion
**Submission Website**: [EasyChair](https://easychair.org/conferences/?conf=usrse2024); please see submission templates below for each type of submission
```

- Training and workforce development
- Maturing and expanding the RSE profession

**Submission Website**: [EasyChair](https://easychair.org/conferences/?conf=usrse2024);
please see submission templates below for each type of submission.

> **Review and Publication**: All contributions will be subject to standard peer

Reviewer: ```suggestion
**Review and Publication**: All contributions will be subject to non-anonymous peer
```

> review for quality, relevance, and novelty, which will determine acceptance and the form of

Reviewer: the form of presentation isn't part of the review process - form is what was submitted.  The reviewers only say yes or no to if it is presented in that format.

presentation. All submissions will be kept confidential prior to publication in
the proceedings. Instructions for camera-ready versions will be sent to authors
of accepted submissions. All submission types will be published as open access
in Zenodo.

Accepted abstracts will be placed on the website ahead of the conference.

> **Questions?** Contact the organizers: [usrse2024@easychair.org](mailto:usrse2024@easychair.org).

Reviewer: ```suggestion
**Questions?** Contact the technical program organizers: [usrse2024@easychair.org](mailto:usrse2024@easychair.org).
```
